dev/make/deps.mkl.mk

```diff
@@ -33,7 +33,7 @@ MKLGPUFPKDIR.libia   := $(MKLGPUFPKDIR)/lib/$(_IA)
 mklgpufpk.LIBS_A := $(MKLGPUFPKDIR.libia)/$(plib)daal_sycl$d.$(a)
 mklgpufpk.HEADERS := $(MKLGPUFPKDIR.include)/mkl_dal_sycl.hpp $(MKLGPUFPKDIR.include)/mkl_dal_blas_sycl.hpp
 
-daaldep.math_backend.incdir := $(MKLFPKDIR.include)
+daaldep.math_backend.incdir := $(MKLFPKDIR.include) $(MKLGPUFPKDIR.include)
```

Comment:
Is this required? Should the `daaldep.math_backend_oneapi.incdir` variable be used when the MKLGPUFPKDIR is needed?

Reply:
It is the question to oneDAL team too. I think it is necessary for MKL backend (to support GPUs). As I can see MKLGPUFPKDIR will be empty if specific env is not set.